packages/atomic/cypress/integration/facets/segmented-facet/segmented-facet.cypress.ts

```diff
@@ -29,22 +36,124 @@ describe('Segmented Facet Test Suites', () => {
     );
   });
 
-  describe.skip('when select a value', () => {
-    describe('verify rendering', () => {});
-    describe('verify analytic', () => {});
+  describe('when selecting a value', () => {
+    const selectionIndex = 2;
+    function setupSelectSegmentedFacet() {
+      setupSegmentedFacet();
+      selectIdleBoxValueAt(selectionIndex);
+    }
+
+    describe('verify rendering', () => {
+      before(setupSelectSegmentedFacet);
+
+      CommonAssertions.assertAccessibility(segmentedFacetComponent);
+      FacetAssertions.assertNumberOfSelectedBoxValues(1);
+      FacetAssertions.assertNumberOfIdleBoxValues(defaultNumberOfValues - 1);
+    });
+    describe('verify analytics', () => {
+      before(setupSelectSegmentedFacet);
+      FacetAssertions.assertLogFacetSelect(field, selectionIndex);
+    });
+  });
+
+  describe('with custom #numberOfValues', () => {
+    const customNumValues = 20;
+    function setupCustomSegmentedFacet() {
+      new TestFixture()
+        .with(
+          addSegmentedFacet({
+            field: 'language',
+            label: 'Language',
+            'number-of-values': customNumValues,
+          })
+        )
+        .init();
+    }
+
+    describe('verify rendering', () => {
+      before(setupCustomSegmentedFacet);
+
+      CommonAssertions.assertAccessibility(segmentedFacetComponent);
+      CommonFacetAssertions.assertLabelContains(
+        SegmentedFacetSelectors,
+        'Language'
+      );
+      FacetAssertions.assertNumberOfIdleBoxValues(customNumValues);
+    });
   });
 
-  describe.skip('with custom #numberOfValues', () => {});
+  describe.skip('when no search has yet been executed', () => {
```

Comment:
Will change from `describe.skip` to `describe` when [the loading animation pr](https://github.com/coveo/ui-kit/pull/2216) is merged